targets/DeployDependencies.proj

```diff
@@ -78,10 +78,13 @@
   <UsingTask TaskName="PrereleaseResolveNuGetPackageAssets" AssemblyFile="$(ToolsDir)net45/Microsoft.DotNet.Build.Tasks.dll"/>
 
   <PropertyGroup>
+    <!-- netstandard is not a runnable framework moniker. If on .net core, target netcoreapp instead, which is runnable -->
```

Comment:
This will be fun to write down :)

For actual, runnable, msbuild deployments, we only target the netcoreapp runtime moniker with the .netstandard1.3 moniker. If we want to deploy msbuild for net46, we'll use the net46 moniker, not .netstandard1.3. If you deploy the .netstandard1.3 flavor of msbuild on net46, you get a full framework msbuild with the feature set of xplat msbuild (no gac, no registry, etc). We never tested that combination.

On the other hand, for API usage (the use case where users create tasks and need to extend ITask) the users won't deploy msbuild for a specific runtime moniker, they will deploy their own task for a specific runtime moniker. Their app just happens to have msbuild in its dependency closure.

This means that:
- we are testing the use case of `get a runnable msbuild deployment for netstandard1.3 on .net core`
- we are not testing the use case of `get a runnable msbuild deployment for netstandard1.3 on net46`, since this does not make much sense (as stated above, you'd get a full framework msbuild with the crossplatform feature set)
- we are not testing the use case of `msbuild is part of the dependency graph of another app, and that app publishes for multiple runtime framework monikers`. We should try and do this :)